grafana/FilesystemDetails.json

```diff
@@ -472,5 +559,5 @@
   },
   "timezone": "",
   "title": "FilesystemDetails",
-  "version": 8
+  "version": 3
```

Comment:
For version lets keep it going up not down.

Reply:
This metric was already supplied by node_exporter

The version number is going to keep resetting every time anyone makes a new installation or someone different contributes a dashboard. Can't really control it.

Reply:
But we can force it to increment